### PR TITLE
Initial build definition

### PR DESCRIPTION
**Pull request by mahilleb-msft**
Add initial build definition.

### CODE REVIEW

File: ci/build.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+#
+# vim:set expandtab shiftwidth=2 tabstop=2:
+#
+# Variables:
+#   SpeechSdkVersion - Speech SDK version to use
+#   SpeechSdkNuGetSource - where to pick up the Speech SDK from ("nuget.org" or "VstsFeed")
```
Comment (wolfma61) [inline ci/build.yml]: >or "VstsFeed" [](start = 77, length = 13)

is this public?

Comment (wolfma61) [reply ci/build.yml]: ok - i see ;) just a url perhaps?

---
In reply to: [194070250](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/10#discussion_r194070250) [](ancestors = 194070250)

Comment (mahilleb-msft) [reply ci/build.yml]: for a VSTS feed, the associated build step actually wants a GUID. We may add additional optinos laters (e.g., pick up from build artifacts of another build).

File: ci/build.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      script: |
+        set -e -x -o pipefail
+        perl ci/patch-samples.pl "$(SpeechSdkVersion)"
+        git diff
+  - task: NuGetCommand@2
+    inputs:
+      command: restore
+      vstsFeed: $(V_VSTS_FEED)
+      includeNuGetOrg: $(V_INCLUDE_NUGET_ORG)
+  - task: VSBuild@1
```
Comment (zhouwangzw) [inline ci/build.yml]: Does it build all samples automatically, or need to specify the sln file one by one?

Comment (mahilleb-msft) [reply ci/build.yml]: The default is to build all solutions, cf. https://docs.microsoft.com/en-us/vsts/pipelines/tasks/build/visual-studio-build?view=vsts

**Pull request merged**